[2024-02-08T08:53:11.713+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6_postgres.insert_into_table scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-08T08:53:11.732+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6_postgres.insert_into_table scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-08T08:53:11.733+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-08T08:53:11.764+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-02-07 00:00:00+00:00
[2024-02-08T08:53:11.770+0000] {standard_task_runner.py:60} INFO - Started process 7355 to run task
[2024-02-08T08:53:11.776+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '6_postgres', 'insert_into_table', 'scheduled__2024-02-07T00:00:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/6_postgres.py', '--cfg-path', '/tmp/tmpmnp0mq9k']
[2024-02-08T08:53:11.779+0000] {standard_task_runner.py:88} INFO - Job 105: Subtask insert_into_table
[2024-02-08T08:53:11.866+0000] {task_command.py:423} INFO - Running <TaskInstance: 6_postgres.insert_into_table scheduled__2024-02-07T00:00:00+00:00 [running]> on host 8c35f9d61d9d
[2024-02-08T08:53:12.103+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cardthp' AIRFLOW_CTX_DAG_ID='6_postgres' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-07T00:00:00+00:00'
[2024-02-08T08:53:12.105+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-02-07', '6_postgres')
        
[2024-02-08T08:53:12.124+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-08T08:53:12.154+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-08T08:53:12.197+0000] {sql.py:450} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-02-07', '6_postgres')
        , parameters: None
[2024-02-08T08:53:12.206+0000] {sql.py:459} INFO - Rows affected: 1
[2024-02-08T08:53:12.229+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=6_postgres, task_id=insert_into_table, execution_date=20240207T000000, start_date=20240208T085311, end_date=20240208T085312
[2024-02-08T08:53:12.271+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-08T08:53:12.301+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-08T08:59:04.067+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6_postgres.insert_into_table scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-08T08:59:04.092+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6_postgres.insert_into_table scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-08T08:59:04.094+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-08T08:59:04.128+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-02-07 00:00:00+00:00
[2024-02-08T08:59:04.135+0000] {standard_task_runner.py:60} INFO - Started process 7792 to run task
[2024-02-08T08:59:04.140+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '6_postgres', 'insert_into_table', 'scheduled__2024-02-07T00:00:00+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/6_postgres.py', '--cfg-path', '/tmp/tmpxxw_slws']
[2024-02-08T08:59:04.143+0000] {standard_task_runner.py:88} INFO - Job 109: Subtask insert_into_table
[2024-02-08T08:59:04.238+0000] {task_command.py:423} INFO - Running <TaskInstance: 6_postgres.insert_into_table scheduled__2024-02-07T00:00:00+00:00 [running]> on host 8c35f9d61d9d
[2024-02-08T08:59:04.434+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cardthp' AIRFLOW_CTX_DAG_ID='6_postgres' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-07T00:00:00+00:00'
[2024-02-08T08:59:04.436+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-02-07', '6_postgres')
        
[2024-02-08T08:59:04.454+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-08T08:59:04.470+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-08T08:59:04.493+0000] {sql.py:450} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-02-07', '6_postgres')
        , parameters: None
[2024-02-08T08:59:04.501+0000] {sql.py:459} INFO - Rows affected: 1
[2024-02-08T08:59:04.525+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=6_postgres, task_id=insert_into_table, execution_date=20240207T000000, start_date=20240208T085904, end_date=20240208T085904
[2024-02-08T08:59:04.594+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-08T08:59:04.621+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
